test/e2e/framework/helper/validation/certificaterequests/certificaterequests.go

```diff
@@ -40,7 +40,7 @@ func ExpectDuration(duration, fuzz time.Duration) func(certificaterequest *cmapi
 		}
 
 		certDuration := cert.NotAfter.Sub(cert.NotBefore)
-		if certDuration > (duration+fuzz) || certDuration < duration {
+		if certDuration > (duration+fuzz) || certDuration < (duration-fuzz) {
```

Comment:
Pebble returns certificates with 1 second less than the configure duration, so I updated this function to allow both lower and higher...which I think makes sense, but there are a couple of places where it is used with 30s fuzz and this change will make those tests much more lenient....perhaps too lenient?
 * https://github.com/search?q=repo%3Acert-manager%2Fcert-manager%20ExpectDuration&type=code

https://github.com/cert-manager/cert-manager/blob/99aded1d141f6c5fd11399f2a1054c50b6f09fcd/test/e2e/suite/issuers/vault/certificate/approle.go#L269-L271